packages/tracking/src/types.ts

```diff
@@ -0,0 +1,167 @@
+export type BaseEvent = {
+  environment: string;
+  user_id: string;
+  hoodname: string;
+  section?: PageSection;
+};
+
+export type ContentEvent = {
+  content_type: string;
+  content_subcategory: string;
+  content_reach: string;
+  content_name: string;
+  content_category: string;
+  content_author: string;
+};
+
+export type PlanInfoEvent = {
+  current_plan: string;
+  currency: string;
+  value: string;
+  item_name: string;
+  transaction_id: string;
+  coupon: string;
+  payment_type: string;
+};
+
+export type PageSection =
```

Comment:
Are these the same for all our apps?

Reply:
yes! since they are used in the GA reports gobally i wanted to type them